manifests/qpid.pp

```diff
@@ -1,8 +1,9 @@
-# Katello Config
+# Katello qpid Config
 class katello::qpid (
-  $client_cert            = undef,
-  $client_key             = undef,
-  $katello_user           = $katello::user
+  $client_cert,
+  $client_key,
+  $katello_user          = $::katello::user,
+  $candlepin_event_queue = $::katello::candlepin_event_queue,
```

Comment:
I don't see this variable in `$::katello`, the exec below was referencing `$::katello::params::candlepin_event_queue`  Maybe candlepin_event_queue could be a param on the qpid class itself?